dev/drivers/scripts/plots/global_ens/jevs_global_ens_headline_gefs_grid2grid_plots.sh

```diff
@@ -21,13 +21,12 @@ export COMPONENT=global_ens
 export VERIF_CASE=grid2grid
 export MODELNAME=gefs
 
-source $HOMEevs/modulefiles/${evs_ver_2d}
+source $HOMEevs/dev/modulefiles/${evs_ver_2d}
```

Comment:
@PerryShafran-NOAA  ... what's going on here? I think this line needs to go.

Reply:
@ShelleyMelchior-NOAA You mean the existence of the global_ens plot job?  

Yeah the line is funky, but this file isn't going into para.

Reply:
@ShelleyMelchior-NOAA Can you fix?